newm-server/src/main/resources/application.conf

```diff
@@ -81,6 +81,12 @@ jwt {
     }
 }
 
+recaptcha {
+    verifyUrl = "https://www.google.com/recaptcha/api/siteverify"
+    secretKey = ${AWS_SECRET_ARN}
+    minScore = 0.5
```

Comment:
Let's make this a db config value with 0.5 as the default instead of hardcoded config.